patches/minecraft/net/minecraft/client/multiplayer/PlayerControllerMP.java.patch

```diff
@@ -1,19 +1,36 @@
 --- ../src-base/minecraft/net/minecraft/client/multiplayer/PlayerControllerMP.java
 +++ ../src-work/minecraft/net/minecraft/client/multiplayer/PlayerControllerMP.java
-@@ -122,6 +122,12 @@
+@@ -14,11 +14,9 @@
+ import net.minecraft.entity.Entity;
+ import net.minecraft.entity.passive.AbstractHorse;
+ import net.minecraft.entity.player.EntityPlayer;
+-import net.minecraft.init.Blocks;
+ import net.minecraft.inventory.ClickType;
+ import net.minecraft.item.ItemBlock;
+ import net.minecraft.item.ItemStack;
+-import net.minecraft.item.ItemSword;
+ import net.minecraft.network.PacketBuffer;
+ import net.minecraft.network.play.client.CPacketClickWindow;
+ import net.minecraft.network.play.client.CPacketCreativeInventoryAction;
+@@ -122,10 +120,16 @@
              }
          }
  
+-        if (this.field_78779_k.func_77145_d() && !this.field_78776_a.field_71439_g.func_184614_ca().func_190926_b() && this.field_78776_a.field_71439_g.func_184614_ca().func_77973_b() instanceof ItemSword)
 +        ItemStack stack = field_78776_a.field_71439_g.func_184614_ca();
 +        if (!stack.func_190926_b() && stack.func_77973_b().onBlockStartBreak(stack, p_187103_1_, field_78776_a.field_71439_g))
+         {
+             return false;
+         }
```

Comment:
This is fine, it just looks a bit weird because a) these are patches of patches and b) the patch looks a bit weird because it is picking up the already existing `return false` block from the if statement that is being removed here and using it for the if statement that forge adds.